cmd/server.go

```diff
@@ -170,7 +181,15 @@ func (o *serverOption) preRunE(cmd *cobra.Command, args []string) (err error) {
 
 		grpcOpts = append(grpcOpts, oauth.NewAuthInterceptor(o.oauthGroup))
 	}
-
+	if o.tls=="tlsup"{
```

Comment:
I prefer to use `bool` instead of `string` here if you just want to have an option to determine if you enable the TLS.